src/lnp/application/message.rs

```diff
@@ -264,7 +264,7 @@ impl Display for Error {
     Clone, PartialEq, Eq, Debug, Display, LightningEncode, LightningDecode,
 )]
 #[lnpbp_crate(crate)]
-#[display(Debug)]
+#[display("open_channel({chain_hash}, {temporary_channel_id}, {funding_satoshis}, {channel_flags})")]
```

Comment:
whenever we do not put all fields into display we need to end it with `, ...`

The same applies for the rest of the messages below

Reply:
Done